R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
I am in the process of creating a lab that walks users through the
Streamflow Stats tool.  When you have a moment, I am hoping you can point
me towards some sources I can read to learn how the stream gages have been
organized into the 4 categories: Gages for Evaluating Streamflow, Gages
Active in 2009, Gages 2 Reference Gages, and Gages Sorted by Period of
Record.  I was thinking it would be good to provide the users with some
explanation on this.

Thank you.

On Thu, May 5, 2016 at 12:04 PM, Jake Nelson <jake.nelson7@gmail.com> wrote:

> Ok.  I will look into that.
>
> Thank you.
>
> On Thu, May 5, 2016 at 11:56 AM, David Blodgett <notifications@github.com>
> wrote:
>
>> In R/Lab 1.Rmd
>> <https://github.com/dblodgett-usgs/NWCEd/pull/5#discussion_r62226122>:
>>
>> > @@ -0,0 +1,197 @@
>> > +---
>>
>> I just tried locally and it's not working like I would expect. I think
>> leaving it as hosted on github is OK for now but we should probably figure
>> it out at some point.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dblodgett-usgs/NWCEd/pull/5/files/1b41e74bc8209dc38bef1127b4a9e7cf9235845c#r62226122>
>>
>
>
>
> --
> Jake Nelson
>



-- 
Jake Nelson